batavia/modules/io.js

```diff
@@ -0,0 +1,217 @@
+    '__file__': 'batavia/modules/io.js',
+    '__name__': 'io',
+    '__package__': ''
+}
+
+io.IOBase = function(args, kwargs) {
+	var types = require('../types')
+
+	PyObject.call(this)
+}
+
+create_pyclass(io.IOBase, 'IOBase')
+
+io.IOBase.__repr__ = function() {
+	return '<IOBase object at 0x99999999>'
+}
+
+// io files will never be equal unless they both reference the same object,
+// since they are types actively reading from the system and maintaining appropriate state
+io.IOBase.prototype.__eq__ = function(other) {
+	return this === other
+}
+
+io.IOBase.prototype.__ne__ = function(other) {
+	return this === other
+}
+
+io.IOBase.prototype.close = function() {
+	if (this.closed) {
+		this.flush()
+		this.closed = true
+	}
+	else {
+		// only the first call will have an effect; multiple close()
+		// operations allowed for convenience.
+	}
+}
+
+io.IOBase.prototype.closed = false
+
+io.IOBase.prototype.fileno = function() {
+    throw new exceptions.UnsupportedOperation.$pyclass(
+        'fileno'
+    )
+}
+
+io.IOBase.prototype.flush = function() {
+}
+
+// since we're in Javascript, we're not in a TTY
+io.IOBase.prototype.isatty = function() {
+	return false
+}
+
+io.IOBase.prototype.readable = function() {
+	return false
+}
+
+io.IOBase.prototype.readline = function() {
+	
+}
+
+io.IOBase.prototype.readlines = function() {
+	
+}
+
+io.IOBase.prototype.seek = function() {
+    throw new exceptions.UnsupportedOperation.$pyclass(
+        'seek'
+    )
+}
+
+io.IOBase.prototype.seekable = function() {
+	return false;
+}
```

Comment:
At column 1: [(indent) Expected indentation of 4 spaces but found 1 tab.](http://eslint.org/docs/rules/indent)